sql/core_clients_daily_v1.sql

```diff
@@ -0,0 +1,76 @@
+WITH
+  numbered_duplicates AS (
+  SELECT
+    * REPLACE(LOWER(client_id AS client_id)),
```

Comment:
Wait, shouldn't this be? or am I misunderstanding something?
```suggestion
    * REPLACE(LOWER(client_id) AS client_id),
```

Reply:
You're right. This is a typo. I force pushed a fix for this.